packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -457,3 +457,83 @@ test('copyWrappedGlobals - copy from prototype too', (t) => {
 
   t.is(Object.keys(target).sort().join(), 'onTheObj,onTheProto,window')
 })
+
+test('getEndowmentsForConfig - allow redefine on properties', (t) => {
+  'use strict'
```

Comment:
this looks meaningful. is it?

Reply:
oops, no. this is AI slop from boilerplate it generated :see_no_evil: